src/tree/index.js

```diff
@@ -99,10 +99,10 @@ class Tree extends Component {
   }
 
   render() {
-    const { searchModeOn } = this.props
+    const { readOnly, searchModeOn } = this.props
 
     return (
-      <ul className={`root ${searchModeOn ? 'searchModeOn' : ''}`} ref={this.setNodeRef}>
+      <ul className={`root ${searchModeOn ? 'searchModeOn' : ''}`} ref={this.setNodeRef} role="listbox" aria-readonly={readOnly} aria-multiselectable={!this.props.simpleSelect}>
```

Comment:
See [previous comment for child/li](https://github.com/dowjones/react-dropdown-tree-select/pull/233/files?file-filters%5B%5D=.js#r271610907)
I think we probably need to use `role="treeitem"` and `role="tree"` on the "parent" instead if !simpleSelect.

We probably should pass disabled down also to when setting aria-readonly?